server/http/src/request.rs

```diff
@@ -85,18 +85,25 @@ mod tests {
 
     const EXAMPLE_SCHEMA: &'static str = "type Query @entity { users: [User!] }";
 
+    fn example_schema() -> Schema {
+        Schema::parse(
+            EXAMPLE_SCHEMA,
+            SubgraphDeploymentId::new("test").unwrap(),
+            IdType::String,
+        )
```

Comment:
We could get away with a smaller diff in this PR if we kept `Schema::parse` the way it is, had it set `IdType::String` internally, and added a separate `Schema::parse_with_id_type` method. I'll leave it to you to decide what you prefer, it's just an idea.